H_min: 96
H_max: 109
S_min: 151
S_max: 255
V_min: 85
V_max: 152
A_min: 1169
A_max: 8961
gaussian_ksize: 0
gaussian_sigma: 0
morph_size: 0
invert_Hue: false
